wai-language-server/src/main.rs

```diff
@@ -0,0 +1,78 @@
+use tower_lsp::jsonrpc::Error;
+use tower_lsp::jsonrpc::Result;
+use tower_lsp::lsp_types::MessageType;
+use tower_lsp::lsp_types::*;
+use tower_lsp::{Client, LanguageServer, LspService, Server};
```

Comment:
Can you merge these imports into a single line? i.e. `use tower_lsp::{jsonrpc::{Error, ...}, lsp_types::MessageType, ...}`.

We should also get rid of the glob import (`use tower_lsp::lsp_types::*`) and just import the things we need.